server/routes/github/github_webhook.py

```
@@ -0,0 +1,318 @@
+    verify_webhook_signature,
+)
+from utils.auth.log_redact import redact_token
+from utils.db.connection_pool import db_pool
+
+logger = logging.getLogger(__name__)
+
+github_webhook_bp = Blueprint("github_webhook", __name__)
+
+
+@github_webhook_bp.route("/webhook", methods=["POST"])
+def github_webhook():
```

Ruff (0.15.12)

[warning] Too many return statements (9 > 6)

(PLR0911)